pkg/shell/indexes.js

```diff
@@ -332,13 +438,89 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
             return listItem;
         }
 
-        var menu = compiled.ordered("menu").map(links);
+        function keyword_relevance(current_best, item) {
+            const native = item.native || false;
+            const coef = item.coef || 0;
```

Comment:
So should the documentation say that the default is 0, not 3?

Reply:
well, thats the thing - default for manifest is 3, but general default is 0. But the documentation is for manifest, therefore I mention there 3